.circleci/config.yml

```diff
@@ -1,8 +1,12 @@
 version: 2.1
 
+slack/notify-on-failure:
+  only_for_branches: "master"
```

Comment:
`notify-on-failure`
Command to notify a Slack channel on failure in the default `master` branch.